app/models/manageiq/providers/ibm_cloud/vpc/cloud_manager/cloud_database.rb

```diff
@@ -3,6 +3,54 @@ class ManageIQ::Providers::IbmCloud::VPC::CloudManager::CloudDatabase < ::CloudD
   supports :delete
   supports :update
 
+  def self.params_for_create(ems)
+    {
+      :fields => [
+        {
+          :component    => 'select',
+          :name         => 'resource_group_name',
+          :id           => 'resource_group_name',
+          :label        => _('Resource Group'),
+          :includeEmpty => true,
+          :isRequired   => true,
+          :options      => ems.resource_groups.map do |rg|
+            {
+              :label => rg.name,
+              :value => rg.name,
+            }
+          end,
+        },
+        {
+          :component    => 'select',
+          :name         => 'database',
+          :id           => 'database',
+          :label        => _('Cloud Database'),
+          :includeEmpty => true,
+          :isRequired   => true,
+          :options      => ["postgresql", "edb", "mysql", "datastax", "mongodb", "elasticsearch", "redis", "etcd"].map do |db|
```

Comment:
I only have access to deploy postgresql cloud databases so not able to confirm for other databases however, the assumption is that they will follow the same format for the `resource_plan_id` https://github.com/ManageIQ/manageiq-providers-ibm_cloud/blob/master/app/models/manageiq/providers/ibm_cloud/vpc/cloud_manager/cloud_database.rb#L13
<img width="1450" alt="Screen Shot 2022-04-12 at 11 46 03 AM" src="https://user-images.githubusercontent.com/48186199/163009274-7e0fde73-d912-4198-a297-84e2a8880910.png">